util.go

```diff
@@ -0,0 +1,213 @@
+package disko
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/apex/log"
+	"github.com/pkg/errors"
```

Comment:
we will also need corresponding go.mod, go.sum changes.

Reply:
ack